tests/test_time.py

```diff
@@ -58,13 +58,22 @@ def test_time_repr():
 
 def test_timer():
```

Comment:
This test case can be removed, as the `Timer` would also be removed.

Reply:
Makes sense, will remove `Timer` and its usages.

Reply:
Because of the `to_next_batch` and `to_next_epoch` methods, I ended up keeping (but refactoring) this test.